File: for loop/evennum.java
/* program to print all the even numbers within the given range.*/

import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        int n,i;
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter any number:");
        n = sc.nextInt();
        
        for(i = 1; i <= n; i++) {

        if(i % 2 == 0) {
            System.out.println(i);
        }
    }
    }
}

File: for loop/evennum.cpp
/* program to print all the even numbers within the given range.*/

#include <iostream>
using namespace std;

int main() {
    int n, i;

    cout<<"Enter any number: ";
    cin>>n;

    for(i = 1; i <= n; i++) {

        if(i % 2 == 0) {
            cout<<i<<endl;
        }
    }

    return 0;
}